server/src/main/java/io/seata/server/session/GlobalSession.java

```diff
@@ -635,9 +652,9 @@ public byte[] encode() {
         } else {
             byteBuffer.putInt(0);
         }
-
         byteBuffer.putLong(beginTime);
         byteBuffer.put((byte)status.getCode());
+        byteBuffer.putInt(active ? 1 : 0);
```

Comment:
It is only a temporary state, why persist, how to ensure compatibility.

Reply:
At the time when #5501 was not merged and this state was not persisted, it would have caused a transaction deranged, but for now it looks like this change can be rolled back

Reply:
done